docs/eq_snapshot.md

```diff
@@ -19,9 +19,154 @@ Example:
         assert 2 + 2 == snapshot(4)
     ```
 
+=== "value changed"
+    <!-- inline-snapshot: outcome-failed=1 -->
+    ```python
+    def test_something():
+        assert 2 + 3 == snapshot(4)
+    ```
+
+=== "--inline-snapshot=fix"
+    <!-- inline-snapshot: fix -->
+    ```python
+    def test_something():
+        assert 2 + 3 == snapshot(5)
+    ```
+
+The value can later be changed with `--inline-snapshot=fix` if the value the snapshot is compared with has changed.
+
+## dirty-equals
+
+It might be, that larger snapshots with many lists and dictionaries contain some values which change frequently and are not relevant for the test.
+
+Example:
+
+=== "original code"
+    <!-- inline-snapshot: outcome-passed=1 outcome-errors=1 -->
+    ```python
+    from inline_snapshot import snapshot
+    import datetime
+
+
+    def get_data():
+        return {
+            "date": datetime.datetime.now(),
+            "payload": "some data",
+        }
+
+
+    def test_function():
+        assert get_data() == snapshot()
+    ```
+
+=== "--inline-snapshot=create"
+    <!-- inline-snapshot: create -->
+    ```python
+    from inline_snapshot import snapshot
+    import datetime
+
+
+    def get_data():
+        return {
+            "date": datetime.datetime.now(),
+            "payload": "some data",
+        }
+
+
+    def test_function():
+        assert get_data() == snapshot(
+            {"date": datetime.datetime(2024, 3, 14, 1, 0), "payload": "some data"}
+        )
+    ```
+
+They might be part of larger data structures and be difficult to normalize.
+
+inline-snapshot tries to change only the values which it has to change in order to make the equality comparison pass.
+This allows to replace parts of the snapshot with [dirty-equals](https://dirty-equals.helpmanual.io/latest/) expressions.
+This expressions are preserved as long as the `==` comparison with them is `True`.
+
+
+Example:
+
+=== "using IsDatetime()"
+    <!-- inline-snapshot: outcome-passed=1 -->
+    ```python
+    from inline_snapshot import snapshot
+    from dirty_equals import IsDatetime
+    import datetime
+
+
+    def get_data():
+        return {
+            "date": datetime.datetime.now(),
+            "payload": "some data",
+        }
+
+
+    def test_function():
+        assert get_data() == snapshot(
+            {
+                "date": IsDatetime(),
```

Comment:
```suggestion
                "date": IsNow(),
```

https://dirty-equals.helpmanual.io/latest/types/datetime/#dirty_equals.IsNow

Reply:
General a good idea, but I can not use `now` in my documentation, because this would give me always a different value for the first example, which would break my tests ( I execute the code snippets in the documentation in my tests)